WiseUnpacker/SabreTools.Serialization.Wrappers/WiseSectionHeader.cs

```diff
@@ -214,49 +218,197 @@ private bool ExtractHeaderDefinedFiles(Stream data, string outputDirectory, bool
 
             // Does output size include the crc32? Doesn't seem to?
 
-            // Extract main MSI file
-            var expected = new DeflateInfo
-            {
-                InputSize = MsiFileEntryLength,
-                OutputSize = MsiFileEntryLength - 4,
-                Crc32 = 0
-            };
-            data.Seek(data.Position - MsiFileEntryLength, 0);
-            if (InflateWrapper.ExtractFile(data, "ExtractedMsi.msi", outputDirectory, expected, false, includeDebug)
-                == ExtractionStatus.FAIL)
+            // Comparing against extractionstatus GOOD for now.
```

Comment:
Please either consolidate or remove comments. Seeing multiple newline-separated comments is a bit strange for this codebase.

Reply:
Should be fixed.